tooling/component-metadata.js

```diff
@@ -30,6 +30,25 @@ const run = () => {
         /* parse the component code to get metadata */
         const data = docgen.parse(code, null, handlers)
 
+        /* remove redundant quotes from default value of type string */
+        Object.values(data.props).forEach(prop => {
+          if (prop.defaultValue) {
+            if (typeof prop.defaultValue.value === 'string') {
+              prop.defaultValue.value = prop.defaultValue.value.replace(/^'(.*)'$/, '$1')
```

Comment:
Do we also need to support double quotes? I'm not sure if `prettier` takes care of this or not, but maybe:

```
/^['"](.*)['"]$/
```

Reply:
Yep, our precommit hook gets rid of double quotes

Reply:
Cool! I'll merge this as-is then.